sources/models/src/aws-k8s-1.15/defaults.d/50-aws-k8s.toml

```diff
@@ -41,6 +41,13 @@ cluster-domain = "cluster.local"
 [settings.metrics]
 service-checks = ["apiserver", "chronyd", "containerd", "host-containerd", "kubelet"]
 
+[services.static-pods]
+configuration-files = []
+restart-commands = ["/usr/bin/static-pods"]
+
+[metadata.settings.kubernetes.static-pods]
+affected-services = ["static-pods"]
+
```

Comment:
We'll need migrations to clean these up on downgrade.

Reply:
Added the migration in https://github.com/bottlerocket-os/bottlerocket/pull/1317/commits/d409af8adac486f904bc4216bdf8328fa2a31bb9